src/main/resources/mustache/typescript-angular/api.service.mustache

```diff
@@ -354,3 +354,4 @@ export class {{classname}} {
 
 {{/operation}}}
 {{/operations}}
+}
```

Comment:
looks like it's wrong but it's actually fine, the triple right brace in operation makes things confuse `{{/operation}}}`.
i think this could be refactored to be:
```
{{/operation}}
}
{{/operations}}
```
to avoid that confuse. wdyt? @mtormento